src/components/AddComment.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from "react";
+import { postCommentByArticleId } from "../api/requests";
+
+import "./AddComment.css";
+
+const AddComment = ({ article_id, comments, setComments }) => {
+  const [comment, setComment] = useState("");
+  const [msg, setMsg] = useState("");
+  const [msgColor, setMsgColor] = useState("");
+  const [isPending, setIsPending] = useState(false);
+
+  const commentDisabled = isPending ? true : false;
+
+  const postNewComment = (e) => {
+    e.preventDefault();
+
+    const newCommentUI = {
+      body: comment,
+      votes: 0,
+      author: "weegembump",
+      article_id: article_id,
+      created_at: Date.now(),
+      comment_id: comments.length + 1,
```

Comment:
```js
comment_id: comments.length + 1
```
This might cause an issue when you come to adding delete functionality. It's good for now but be aware of how it could lead to duplicates when the id's are not necessarily sequential. 

https://www.npmjs.com/package/uuid is a common solution if you need to create unique id's in the future 👍